backend/server.js

```diff
@@ -1,27 +1,114 @@
 const express = require("express");
 const cors = require("cors");
 const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
 
 require("dotenv").config();
 
 const app = express();
 const port = process.env.PORT || 5005;
 
+// Middleware
 app.use(cors());
 app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.set("view engine", "ejs");
 
+// Connect to MongoDB
 // const uri = process.env.ATLAS_URI;
```

Comment:
Uncomment this and delete lines 19 and 20 since you are revealing the URI code publicly